modules/nf-core/gatk4spark/markduplicates/tests/main.nf.test

```diff
@@ -119,40 +119,123 @@ nextflow_process {
         }
 
         then {
+            assert process.success
             assertAll(
-                { assert process.success },
                 { assert snapshot(process.out).match() }
             )
         }
-
     }
 
-    test("stub") {
+    test("sarscov2 - bam -stub") {
+        options "-stub"
         config "./bam.config"
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test', single_end:false ], // meta map
+                    file(params.test_data['sarscov2']['illumina']['test_paired_end_sorted_bam'], checkIfExists: true)
+                ]
+                input[1] = file(params.test_data['sarscov2']['genome']['genome_fasta'], checkIfExists: true)
```

Comment:
I'm also wondering if it is really necessary to populate all the stub test with actual file that will be needed to be downloaded if we don't use them at all in the end...

Reply:
that's a valid question indeed

Reply:
let's take that to the next maintainer meeting